### DISCUSSION TITLE
useKeyPress improvement or am I doing it wrong?

### DISCUSSION THREAD

**Discussion by raarts** [xyflow/xyflow#2074] [Ideas]
Edit: tried with 10.1.0 

Currently I use your useKeyPress hook to track the Ctrl key for Crtl-Click multiple selection.
I also use it for Ctrl-Z, undo actions.

If I use multiple Ctrl-Z in a row, I find that I can't keep the Ctrl-key pressed and repeatedly press the 'z' key for multiple consecutive undo actions, I need to release the Ctrl key between every undo action.
This is a bit un-intuitive. Am I using it wrong? Is this a bug? Or a feature request?

(this is unrelated to react-flow by the way).

**Comment by standard-software** [comment]
In my case.
const keyPressCopy = useKeyPress(['Meta+c', 'Control+c']);

I am having the exact same problem. The workaround is.
Is there a workaround?
